src/components/TransactionsInterpreter/helpers/getDenominatedValue.ts

```diff
@@ -1,15 +1,15 @@
-import { UITransactionType } from './types';
-import { denominate } from 'utils';
-import { DenominationConfig } from '../TransactionsInterpreter';
 import { defaultNetwork } from 'reduxStore/slices';
+import { denominate } from 'utils/operations/denominate';
+import { DenominationConfig } from '../TransactionsInterpreter';
+import { UITransactionType } from './types';
 
 export function getDenominatedValue(
```

Comment:
pls move this function in utils/operations because it's used in other projects too

Reply:
This function is only a wrapper that is used only by the transaction interpreter. There is a function called `denominate` which is already in utils/operations (used by this one)